File: Python programming/Switch1.py
day = "thursday"
match (day):
            case "monday":
                print("Basketball")
                
            case "tuesday":
                print("Tennis")   

            case "wednesday":
                print("Basketball")
              
            case "thursday":
                print("Cricket")
               
            case "friday":
                print("Fotball")
                
            case "saturday":
                println("Excerise")
               
            case "sunday":
                println("Holiday")
            
            case _:
                print("invalid "+day)
                
      

File: java programming/Switch1.java
public class Switch1 {
    public static void main(String[] args) {

        String day = "thursday";
        switch (day) {
            case "monday":
                System.out.println("Basketball");
                break;
            case "tuesday":
                System.out.println("Tennis");
                break;

            case "wednesday":
                System.out.println("Basketball");
                break;

            case "thursday":
                System.out.println("Cricket");
                break;

            case "friday":
                System.out.println("Fotball");
                break;

            case "saturday":
                System.out.println("Excerise");
                break;

            case "sunday":
                System.out.println("Holiday");
                break;

            default:
                System.out.println("invalid "+day);
                break;
        }
    }
}
